<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TRTimeOperator</class>
 <widget class="QWidget" name="TRTimeOperator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TRTimeOperator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-character-map" resource="../qrc/icon.qrc">
    <normaloff>:/img/logo.ico</normaloff>:/img/logo.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::East</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Operation</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_Theraphy">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>300</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Theraphy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LeavingRoom">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Leaving Room</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>21</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PatentImaging">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Imaging/Position</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>360</y>
       <width>21</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PatentComeIn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Patient Come in</string>
     </property>
     <property name="icon">
      <iconset theme="address-book-new"/>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>240</y>
       <width>21</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/icon.qrc"/>
 </resources>
 <connections/>
</ui>
